numpyro/distributions/kl.py

```diff
@@ -126,6 +133,19 @@ def kl_divergence(p, q):
     return 0.5 * (var_ratio + t1 - 1 - jnp.log(var_ratio))
 
 
+@dispatch(Beta, Beta)
+def kl_divergence(p, q):
+    # From https://en.wikipedia.org/wiki/Beta_distribution#Quantities_of_information_(entropy)
+    a, b = p.concentration1, p.concentration0
+    alpha, beta = q.concentration1, q.concentration0
+    a_diff = alpha - a
+    b_diff = beta - b
+    t1 = betaln(alpha, beta) - betaln(a, b)
```

Comment:
When I compare with wiki I get `\alpha=alpha`, `\beta=beta`, `\alpha'=a` and `\beta'=b` so `t1 = betaln(a,b) - betaln(alpha,beta)`.